js/bithumb.js

```diff
@@ -1031,6 +1009,18 @@ module.exports = class bithumb extends Exchange {
         };
     }
 
+    fixCommaNumber (numberStr) {
+        // some endpoints need this https://github.com/ccxt/ccxt/issues/11031
+        if (numberStr === undefined) {
+            return undefined;
+        }
+        let finalNumberStr = numberStr;
+        while (finalNumberStr.indexOf (',') !== -1) {
```

Comment:
@ttodua, I'm a bit confused, why do we need this loop?

Isn't the `replace` function to replace all occurrences on the first call, or am I missing something?

```
const x = "122,33,44,55"
const y = x.replace(",", "") // y = 122334455

Reply:
actually it doesn't, that would be replaceAll or replace (/regex/g)

Reply:
yes, as carlo said.  (we dont have regex codes transpilable, so had to use that approach). alternatively, we might add 'replaceG/replaceAllOccurences' method in base. thoughts?

Reply:
@ttodua @frosty00  yeah, my bad what I said is only valid for PY and PHP, not JS

![image](https://user-images.githubusercontent.com/43336371/182575943-8a15ff98-7da3-4004-94a5-d913161f3aee.png)
![image](https://user-images.githubusercontent.com/43336371/182576272-84d8a5ea-8008-4e37-ad3c-246b75d351c2.png)